docs/components/MessagingAttachmentView.jsx

```diff
@@ -83,7 +82,8 @@ export default class MessagingAttachmentView extends React.PureComponent {
         icon: <FileAttachmentIcon fileType={"audio"} isUpload uploadError />,
         onClickAttachment: () => console.log("clicked!"),
         onRemoveAttachment: () => console.log("removed!"),
-        title: "File cannot exceed 10 MB",
+        title: "My cool file.txt",
```

Comment:
This kind of makes me think `title` should actually be optional, since it's unnecessary when there's an error? I think ideally we'd have an Error version of this component (a la AnnouncementBubble's structure), but that seems like overkill just for this.

We can add it in Milestone 2, when we'll be adding the full-bleed image preview version of this component.

Reply:
I was thinking the same